health/health.d/geth.conf

```diff
@@ -0,0 +1,25 @@
+ template: chainhead_diff_between_header_block
+       on: geth.chainhead
+    class: Workload
+     type: ethereum_node
+component: Geth
```

Comment:
What do you think about type: Etherium, component: Node?

Reply:
That doesn't make sense. Let's keep it that way

Reply:
> Type can be used to indicate the broader area of the system that the alarm applies to. For example, under the general Database type, you can group together alarms that operate on various database systems, like MySQL, CockroachDB, CouchDB etc.

> Ethereum is a distributed network of computers running software (known as nodes) that can verify blocks and transaction data. You need an application, known as a client, on your computer to "run" a node.

For me, it looks like the general type is Etherium. And specific component is Node (client, if it is running it makes your server a Node => part of the Etherium network).

Reply:
Just thinking out loud 🤷‍♂️

Reply:
[src](https://ethereum.org/en/developers/docs/nodes-and-clients/)

> "Node" refers to a running piece of client software. A client is an implementation of Ethereum that verifies all transactions in each block, keeping the network secure and the data accurate.